### PR TITLE
Adding details about Encore dev-server and https

### PR DESCRIPTION
**Pull request by weaverryan**
Hi!

See: https://github.com/symfony/cli/issues/93

Just some missing docs on use the `dev-server` with the Symfony binary.

### CODE REVIEW

File: frontend/encore/dev-server.rst [hunk 1]
```diff
@@ -17,6 +17,24 @@ Twig shortcuts (or are :ref:`processing your assets through entrypoints.json <lo
 in some other way), you're done: the paths in your templates will automatically point
 to the dev server.
 
+Enabling https & the Symfony Web Server
```
Comment (fabpot) [inline frontend/encore/dev-server.rst:20]: HTTPS everywhere instead of https?

File: frontend/encore/dev-server.rst [hunk 2]
```diff
@@ -17,6 +17,24 @@ Twig shortcuts (or are :ref:`processing your assets through entrypoints.json <lo
 in some other way), you're done: the paths in your templates will automatically point
 to the dev server.
 
+Enabling HTTPS & the Symfony Web Server
+---------------------------------------
```
Comment (tucksaun) [inline frontend/encore/dev-server.rst:21]: maybe
```suggestion
Enabling HTTPS using the Symfony Web Server
-------------------------------------------
```

?

**Comment by wouterj** [comment]
Not related with this PR directly, but what if encore would automatically configure this for us using the [`devServer.pfx`](https://webpack.js.org/configuration/dev-server/#devserverpfx) config?

**Comment by weaverryan** [comment]
Yea, you're totally right. I'll check into possibly automatically detecting the `.p12` file and configuring the `pfx` for the user in Encore (unless they have configured it themselves). If that doesn't feel right or doesn't work, I'll ask about a possible Encore specific command.

We should merge this for now though - I'll follow-up on that other solution.

**Review by fabpot — APPROVED** [review]

**Review by wouterj — APPROVED** [review]

**Pull request closed without merging**